docs/windows.md

```diff
@@ -1,142 +1,141 @@
 
-To actually build go-ipfs, first go to the cmd/ipfs directory:
+|Command|Explanation|
+| ---: | :--- |
+|`SET PATH=%PATH%;\cygwin64\bin`           |Add Cygwin's bin to PATH; Defaults to: (\cygwin64\bin)|
+|`mkdir %GOPATH%\src\github.com\ipfs`<br/>`cd %GOPATH%\src\github.com\ipfs`<br/>`git clone https://github.com/ipfs/go-ipfs.git`       |Fetch / Update `go-ipfs` source|
+|`cd %GOPATH%\src\github.com\ipfs\go-ipfs` |Change to `go-ipfs` directory|
+|`make install`                            |Build and install to `%GOPATH%\bin\ipfs.exe`|
+|`%GOPATH%\bin\ipfs.exe version --all`     |Test the built binary|
 
+**Tip**: You can permanently append the Cygwin bin to your `PATH` so that on subsequent builds you can simply run
 ```
-cd cmd\ipfs
+cd %GOPATH%\src\github.com\ipfs\go-ipfs
+make install
 ```
-
-Then get the current Git commit:
-
+This can be achieved with the `setx` command: 
 ```
-git rev-parse --short HEAD
+SETX PATH %PATH%;\cygwin64\bin
 ```
 
-It will output a small number of hex characters that you must pass to
-the actual build command (replace XXXXXXX with these characters):
+## Minimal
+While it's possible to build `go-ipfs` with `go` alone, we'll be using `git` and `gx` for practical source management.  
+You can use whichever version of `git` you wish but the Windows builds at <https://git-scm.com> are recommended. `git` must be in your `PATH` for `go get` to recognize and use it.
+
+### `gx`
+You may install prebuilt binaries for [`gx`](https://dist.ipfs.io/#gx) & [`gx-go`](https://dist.ipfs.io/#gx-go) if they're available for your platform.
+Alternatively you can build them from source and install them to `%GOPATH%\bin` by running the following:
 
 ```
-go install -ldflags="-X "github.com/ipfs/go-ipfs/repo/config".CurrentCommit=XXXXXXX"
+go get -u github.com/whyrusleeping/gx
+go get -u github.com/whyrusleeping/gx-go
 ```
 
-After that ipfs should have been built and should be available in
-"%GOPATH%\bin".
-
-You can check that the ipfs you built has the right version using:
+### `go-ipfs`
 
 ```
-ipfs version --commit
+SET PATH=%PATH%;%GOPATH%\bin
+go get -u -d github.com/ipfs/go-ipfs
+cd %GOPATH%/src/github.com/ipfs/go-ipfs
+gx --verbose install --global
+cd cmd\ipfs
 ```
+We need the `git` commit hash to be included in our build so that in the extremely rare event a bug is found, we have a reference point later for tracking them. We'll ask `git` for it and store it in a variable. Unfortunately the syntax for the next command is different depending on how it's executed. If you're on the interactive command line vs executing from inside the batch/cmd interpreter. Use the one that applies to you.  
```

Comment:
The first sentence needs both “**a** bug is found” and “tracking **them**” to be the same singular or plural.

> If you're on the interactive command line vs executing from inside the batch/cmd interpreter.

This isn’t a full sentence. It would probably read better combined with the previous sentence, like:

> The syntax for the next command is different depending whether you're on the interactive command line or executing it from inside the batch/cmd interpreter.